test/ZendeskApi_v2.Test/OrganizationTests.cs

```diff
@@ -370,5 +543,113 @@ public async Task CanCreateUpdateAndDeleteMultipeOrganizationsAsync()
             await api.Organizations.DeleteOrganizationAsync(res1.Organization.Id.Value);
             await api.Organizations.DeleteOrganizationAsync(res2.Organization.Id.Value);
         }
+
+        [Test]
+        public async Task CanCreateAndDeleteMultipleOrganizationsUsingBulkApisAsync()
+        {
+            var createJobStatus = await api.Organizations.CreateMultipleOrganizationsAsync(new[]
+            {
+                new Organization
+                {
+                    Name = "Bulk Create Org 1"
+                },
+                new Organization
+                {
+                    Name = "Bulk Create Org 2"
+                }
+            });
+
+            Assert.That(createJobStatus.JobStatus.Status, Is.EqualTo("queued"));
+            JobStatusResponse job;
+            do
+            {
+                Thread.Sleep(1000);
```

Comment:
Please for async methods use an async sleep.
```suggestion
                 await Task.Delay(1000);
```

Reply:
I _think_ I just copied this code, so I'll do a quick check in other places